src/InfoViz/Native/MutualInformationDiagram/index.js

```diff
@@ -964,6 +964,10 @@ function informationDiagram(publicAPI, model) {
       });
 
     model.subscriptions.push(model.mutualInformationDataSubscription);
+
+    if (model.provider.setMutualInformationParameterNames) {
```

Comment:
You mean to say that the check whether the provider is a MutualInformationProvider should be good enough?